File: CSharp/Controls/LogViewer.Wpf/Converters/ChangeColorTypeConverter.cs
using System;
using System.Globalization;
using System.Windows.Data;
using System.Windows.Media;
using SysDrawColor = System.Drawing.Color;

namespace LogViewer.Wpf.Converters;

public class ChangeColorTypeConverter : IValueConverter
{
    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
    {
        SysDrawColor sysDrawColor = (SysDrawColor)value;
        return new SolidColorBrush(Color.FromArgb(
            sysDrawColor.A,
            sysDrawColor.R,
            sysDrawColor.G,
            sysDrawColor.B));
    }

    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
        => throw new NotImplementedException();
}

File: VB/Controls/LogViewerVB.Avalonia/Converters/ChangeColorTypeConverter.vb
Imports System.Globalization
Imports Avalonia.Data.Converters
Imports Avalonia.Media
Imports SysDrawColor = System.Drawing.Color

Namespace Converters

    Public Class ChangeColorTypeConverter : Implements IValueConverter

        Public Function Convert(value As Object, targetType As Type, parameter As Object, culture As CultureInfo) As Object Implements IValueConverter.Convert

            Dim sysDrawColor As SysDrawColor = DirectCast(value, SysDrawColor)
            Return New SolidColorBrush(Color.FromArgb(
                sysDrawColor.A,
                sysDrawColor.R,
                sysDrawColor.G,
                sysDrawColor.B))

        End Function

        Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
            Throw New NotImplementedException
        End Function

    End Class

End Namespace